docs/Manual/00 Introduction.md

```diff
@@ -2,40 +2,40 @@
 [//]: # (Version: v1.2.1)
 
 # Introduction
-Accera is a programming model, a domain-specific programming language embedded in Python (eDSL), and an optimizing cross-compiler for compute-intensive code. Accera currently supports CPU and GPU targets and focuses on optimization of nested for-loops.
+Accera is a Python-based embedded domain-specific programming language (eDSL) that enables cross-compiler optimization for compute-intensive codes. Currently, optimization of nested for-loops is the primary focus of Accera while supporting CPU and GPU targets.
 
-Writing highly optimized compute-intensive code in a traditional programming language is a difficult and time-consuming process. It requires special engineering skills, such as fluency in Assembly language and a deep understanding of computer architecture. Manually optimizing the simplest numerical algorithms already requires a significant engineering effort. Moreover, highly optimized numerical code is prone to bugs, is often hard to read and maintain, and needs to be reimplemented every time a new target architecture is introduced. Accera aims to solve these problems.
+Optimization of compute-intensive code in a traditional programming language is not only difficult and time-consuming, but manual optimization of simplest numerical algorithms is also a nightmare even for an engineer who has an advanced understanding of computer architecture and fluency in Assembly Language. Even with all these efforts, implemented code is prone to critical bugs and requires extensive engineering effort for maintenance. Accera aims at resolving all these issues by providing optimized solutions for compute-intensive algorithms that are highly efficient, readable, and maintainable. 
```

Comment:
instead of "nightmare" maybe we say something like "significant undertaking" which is slightly less dramatic :)